src/SDKs/dirs.proj

```diff
@@ -60,6 +61,8 @@
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Automation\**\*Tests.csproj" ProjectType="Test"/>
     
 		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Batch\Management\**\*Tests.csproj" ProjectType="Test"/>
+		<ProjectToBuild Include="$(MSBuildThisFileDirectory)Batch\DataPlane\Client\Tools\ObjectModelCodeGeneration\ObjectModelCodeGenerator\ObjectModelCodeGenerator.csproj" ProjectType="Test"/>
```

Comment:
Agree, lets chat.

For _right now_ I've disabled all the projects that don't work in `Directory.Build.props`, but I would like to get them enabled again. Let's chat today about what we can do.